ntirpc/rpc/svc.h

```diff
@@ -318,6 +320,18 @@ struct svc_xprt {
 	} xp_pktinfo;
 };
 
+#define XPRT_FMT "xprt: [ptr = {},flags = {},fd = {},type = {},refcnt = {}]"
+#define XPRT_VARS(_xprt) (_xprt), (_xprt)->xp_flags, (_xprt)->xp_fd, \
+	(_xprt)->xp_type, (_xprt)->xp_refcnt
+
+#define XPRT_AUTO_TRACEPOINT(_xprt, event, log_level, format, ...) \
+	NTIRPC_AUTO_TRACEPOINT(xprt, event, log_level, \
```

Comment:
Should this 'xprt' be '_xprt'?  It will likely compile, since we'll almost always have a symbol named 'xprt' in the function, but it seems wrong.

Reply:
No, actually. The first parameter is the provider name, which is an auto-generated object. _xprt refers to the xprt object you want to log, while this refers to the name of the event that will be presented when you look at the traces.